ht-8/index.js

```diff
@@ -0,0 +1,31 @@
+function initCollapse(){
+  const buttons = Array.from(document.querySelectorAll(`[data-toggle="collapse"]`));
+  buttons.forEach(button => {
+    button.addEventListener('click', () => {
+      const cardId = button.getAttribute('href') || button.dataset.target;
+      collapseShow(cardId, button);
+    })
+  });
+  const collapseShow = (cardId, button) => {
+    const parentButton = button.parentElement.parentElement.parentElement;
```

Comment:
You probably should learn about the [.closest()](https://developer.mozilla.org/en-US/docs/Web/API/Element/closest) method.